sycl/doc/extensions/supported/sycl_ext_intel_esimd/README.md

```diff
@@ -120,37 +125,5 @@ Currently, compilation of programs with `invoke_simd` calls requires a few addit
 # and callee in the same module.
```

Comment:
This comment applies to line 87 above:

> More details are available in [invoke_simd spec](../sycl_ext_oneapi_invoke_simd.asciidoc)

This link is now broken because you moved the location of the "README.md" file.  The "sycl_ext_oneapi_invoke_simd" spec is in the "experimental" folder.  Check for other cases where you have relative file references, which may have a similar problem.

Reply:
Thank you. I fixed this issue + one more in another place. d6b2ac049189639af1f1a13eceaeb2a38b1235e4